src/main/java/nl/avans/drivio/repository/HydrogenCarRepository.java

```diff
@@ -0,0 +1,10 @@
+package nl.avans.drivio.repository;
+
+import nl.avans.drivio.model.Car;

```

Comment:
Is het Carmodel hier nog nodig als import?

Reply:
Ehm als het goed is niet. Ik ga vanavond even kijken of (en waar) deze gebruikt wordt, als dit nergens is haal ik hem eruit

Reply:
Ik heb de wijziging uitgevoerd (zie commit [[Feat: Changed the requested changes from PR#25](https://github.com/leonvsc/Drivio/pull/25/commits/8fa43389fecb9ea25438afd652b9d161e75421ff))](https://github.com/leonvsc/Drivio/pull/25/commits/8fa43389fecb9ea25438afd652b9d161e75421ff)